source/type/type_utils.cpp

```diff
@@ -82,4 +82,22 @@ pbnavitia::RTLevel to_pb_realtime_level(const navitia::type::RTLevel realtime_le
             throw navitia::exception("realtime level case not handled");
     }
 }
+
+namespace type {
+
+pbnavitia::DirectionType get_direction_type(std::string& direction_type) {
+    if (direction_type == "all") {
+        return pbnavitia::DirectionType::ALL;
+    } else if ((direction_type == "forward") || (direction_type == "clockwise") || (direction_type == "inbound")) {
+        return pbnavitia::DirectionType::FORWARD;
+    } else if ((direction_type == "backward") || (direction_type == "anticlockwise")
+               || (direction_type == "outbound")) {
+        return pbnavitia::DirectionType::BACKWARD;
+    } else {
```

Comment:
maybe for an empty value, but if it ask `foo` as a direction type I expect an error

Reply:
I have seen with @pbougue that for strange case like `direction_type=""` or `direction_type="poo"`, we don't want to filter and we don't want to crash the request.
But If everybody is not agree I'll change it again :tada